pkgs/tools/system/osquery/default.nix

```diff
@@ -106,5 +106,6 @@ stdenv.mkDerivation rec {
     license = licenses.bsd3;
     platforms = platforms.linux;
     maintainers = with maintainers; [ cstrahan ma27 ];
+    broken = true;
```

Comment:
What exactly broke `osquery`'s build here? And have you checked if there are some upstream patches to work around the problem?

Reply:
actually, the osquery nixpkg is quite outdated. a plugin of osquery ( in the current source ) depends on rocksdb-lite. the plugin expects the lite version of rocksdb to be linkable via -lrocksdb_lite. but since i didn't have time and i didn't understand the osquery nixpkg ( looks quite complicated ). hence, i decided to mark osquery broken. i need to be repackaged anyways.

Reply:
@flokli maybe regarding osquery?

Reply:
In fact I'm on of the maintainers of osquery, that's the original reason why I asked.
In the end @magenbluten is right, osquery is behind upstream by at least one minor. I tried to do the bump last month (hoping that this would solve several issues), but I faced some hard-to-solve erors back then. I hope that I have sufficient time soon to do the bump.

Reply:
@Ma27 if you could disable the rocksdb plugin or `fix` the CMakefile to link against `-lrocksdb` and not `-lrocksdb_lite` everything should work as expected. However, the rocksdb plugin says it wants rocksdb 5.7.2 afair, this is also outdated. the current release of osquery is 3.3.2 whereas the nixpkg version is 3.2.9.

Reply:
Hmm I see. Admittedly I'm slightly hesitant to patch the CMake files even further as it diverges pretty much from upstream already.

Due to `osquery` being slightly outdated and causing issues with further packages I guess that it should be fine for now to mark it as broken. I already have a local branch where I tried to bump/repackage osquery, but it's far from done. I hopefully have sufficient time for some more hacking next week, I can give this a try then :)